apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -80,19 +82,24 @@ class PiskelEditor extends React.Component {
   }
 
   UNSAFE_componentWillReceiveProps(newProps) {
-    if (newProps.selectedAnimation !== this.props.selectedAnimation) {
+    if (
+      newProps.currentAnimations[newProps.currentAnimationType] !==
+      this.props.currentAnimations[this.props.currentAnimationType]
```

Comment:
nit: this conditional and the one we changed below are a little hard to read because of their verbosity. but destructuring the data we need might result in some weird variables names:

```javascript
// newCurrentAnimations vs oldCurrentAnimations? kind of weird names...
const { currentAnimations: newCurrentAnimations } = newProps
const { currentAnimations: oldCurrentAnimations } = this.props
```

thoughts? maybe we leave as-is?

Reply:
Thanks to your other suggestion, I was able to simplify this back down to `newProps.currentAnimation !== this.props.currentAnimation`! 30061ac21e17c486f675b5c884c927dba4c7aab6